tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.yaml

```diff
@@ -5,27 +5,38 @@ context:
         - author:
             name: Nick Lee
             role: Designer
+        - author:
```

Comment:
Again it might be best to leave these for @chris-lawton to make merging simpler

Reply:
Chatted with Victor on Slack - i'll update the pattern lib changes on the FE branch